Dockerfile

```diff
@@ -15,4 +15,4 @@ RUN addgroup --gid 1001 --system app && \
   chmod 770 /app
 USER app:app
 COPY --chown=app:app --from=build /app/target/app.jar ./
-CMD java -jar app.jar
+CMD java -XX:+UseZGC -Xss10m -jar app.jar
```

Comment:
sliter metadata med stackoverflow?

Reply:
@valosnah tipper du må øke hvor mye minne som er tilgjengelig for poden, dette vil sannsynligvis øke minnebruken drastisk.

Reply:
Ja, den sliter med stackoverflow. I det den kjører validering internt i jena, er det en lang rekke med rekursive funksjoner, som kan kortslutte noen ganger. Vi har testet det å øke stack size med Hans Olav og det funker like fint uten å konsummere ektra ressurser.

Reply:
Ser ut til å være 'litt' høye topper i staging atm:

![Screenshot from 2021-03-23 08-34-25](https://user-images.githubusercontent.com/50194012/112110106-be6a2900-8bb2-11eb-861b-96f33a1480b3.png)

Reply:
Damn, jeg så slik oppførelse i lokalt da jeg brukte ny GC, faktisk.

Reply:
Med ZGC burde det gå bra for poden sin del, så lenge noden har nok minne tilgjengelig. Men den vil stå i fare for å plutselig bli kastet ut av noden.

Reply:
Den økte lokalt når du gikk over til ny GC uten å øke Xss?

Reply:
Kanskje det bare er ZGC som er litt mer liberal med tildeling av minne, men @valosnah bør uansett øke minnetildelingen i helm.

Reply:
Sjekk disse bildene:

Uten ZGC:
<img width="1780" alt="Screenshot 2021-03-23 at 08 45 09" src="https://user-images.githubusercontent.com/29962353/112111894-1ace4800-8bb5-11eb-9b51-1ce932c9b10d.png">

Med ZGC:
<img width="1782" alt="Screenshot 2021-03-23 at 08 51 34" src="https://user-images.githubusercontent.com/29962353/112111915-23bf1980-8bb5-11eb-8b38-68bb1261f081.png">